src/main/kotlin/it/pagopa/checkout/authservice/client/oneidentity/OneIdentityClient.kt

```diff
@@ -0,0 +1,38 @@
+package it.pagopa.checkout.authservice.client.oneidentity
```

Comment:
```suggestion
package it.pagopa.checkout.authservice.clients.oneidentity
```

bikeshedding: use plural for package names as done for other packages

Reply:
Good point, this particular package was always singular in other repos I referenced but it should have been plural all along, addressed with [b24ed8f](https://github.com/pagopa/pagopa-checkout-auth-service/pull/12/commits/b24ed8f66fac80a9fed94b25add28496e815e8f6) ✅